test/videoblocks.js

```diff
@@ -0,0 +1,140 @@
+const { expect } = require('chai');
+const { getCredentials, nockBack } = require('./util');
+const credentials = getCredentials();
+const { videoblocks } = require('..')(credentials);
+
+describe('service: videoblocks', function () {
+  this.timeout(5000);
```

Comment:
Is this so that when we are running in fixture generation mode, then you we don't time out?

Reply:
@timlesallen yes